helm/reana/values.yaml

```diff
@@ -13,7 +13,7 @@ eos:
 
 workspaces:
   paths:
-    - "/var/reana"
+    - /var/reana: /var/reana
```

Comment:
I personally find key value pairs for defining workspace paths a bit counterintuitive. I guess it could easily lead to some deployment errors if administrator would specify `/var/eos:/eos` in one line without the space in between. I would propose to use strings here as before. WDYT?